src/execution/index.ts

```
@@ -0,0 +1,79 @@
+import { ExecutionConfig, ExecutionResult } from './execution.types';
+import { IExecutionService } from './execution.interface';
+import fs from "fs";
+import path from "path";
+import { ReportGenerator } from '../report/report-generator';
+
+class ExecutionService implements IExecutionService {
+  async execute(config: ExecutionConfig, directory: string = path.join(__dirname, '../../examples')): Promise<ExecutionResult> {
```

ESLint

[error] Replace `config:·ExecutionConfig,·directory:·string·=·path.join(__dirname,·'../../examples')` with `⏎····config:·ExecutionConfig,⏎····directory:·string·=·path.join(__dirname,·'../../examples'),⏎··`

(prettier/prettier)